examples/isaacsim-sim/README.md

```diff
@@ -0,0 +1,132 @@
+# dora-rs & isaacsim
+
+![dora_isaacsim_act](./assets/dora_isaacsim_act.png)
+
+## Preparation
+
+1. Clone this repository;
+2. Download model parameters `policy_best.ckpt` and `dataset_stats.pkl` to `node-hub/policy-act/policy_act/assets/ckpt/`
+   + Here is an example [act_model](https://drive.google.com/drive/folders/1F1GcguN-Zlrq22FOAqiBiZu6iCwIX6EC?usp=sharing).
+3. Download [isaacsim](https://docs.isaacsim.omniverse.nvidia.com/4.5.0/installation/download.html);
+4. Download [scenary](https://drive.google.com/drive/folders/1wZEysysqdKG0UYsZLl51Rdi7MFiYbfbP?usp=sharing) and unzip to `node-hub/dora-isaacsim/dora_isaacsim/assets/`
```

Comment:
Is there a more official links to scenary? i don't know how rmaintainable it is to use some drive google to keep track of code.